spec/controllers/dashboard_controller_spec.rb

```diff
@@ -52,4 +54,33 @@
       end
     end
   end
+
+  describe "GET /kubectl-config" do
+    it "gets redirected if not logged in" do
+      get :kubectl_config
+      expect(response.status).to eq 302
+    end
+
+    context "HTML rendering" do
```

Comment:
[Minor] I think it would be better if we used something like "format.yaml do" to return the correct mime type since we are not rendering HTML after all.